src/test-run/index.js

```diff
@@ -228,5 +240,5 @@ ServiceMessages[CLIENT_MESSAGES.ready] = function (msg) {
         }
     }
 
-    return new Promise((resolve, reject) => this.pendingRequest = { resolve, reject });
+    return new Promise((resolve, reject) => this.pendingRequest = { statusId: driverStatus.id, resolve, reject });
```

Comment:
BTW, we need a comment which will describe why we need it and which scenarios it handles